src/doc/using_visit/MakingItPretty/Color_tables.rst

```diff
@@ -523,16 +520,20 @@ Tags must now only be comprised of alphanumeric characters, whitespace, dashes,
 Searching for Color Tables
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Another option for finding the right color table is to enable searching. 
-With this option, users can type a search term into the **Name** text box, and only color tables that have a name containing that term will appear in the color table name table.
+Another option for finding the right color table is to use the search bar.
+With this option, users can type a search term into the **Search** text box, and only color tables that have a name containing that term will appear in the color table name table.
+Prior to VisIt_ 3.4, the **Name** text box doubled as the search bar, and users had to check the box for it to the right of the **Name** text box.
 
-To enable searching, check the box for it to the right of the **Name** text box.
-While searching is enabled, searching will happen automatically each time the **Name** text box is edited.
-If a selection is made from the color table name table, searching will temporarily pause.
-However, as soon as the **Name** text box is edited once more, searching will resume as before.
-The only way to stop searching is to disable it via the check box.
+As of VisIt_ 3.4, searching is always on.
+Users need only edit the search bar contents to use it.
+To disable searching, simply delete all text in the search bar.
```

Comment:
528 and 529 here are fine to include. I can't recall, does an empty search box have greyed out text that hints at what to do? like Enter search text here?

Reply:
Thats fine too. I see that kind of thing, for example, all over GitHub's browser GUI.